server/go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/mattn/go-isatty v0.0.17 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
What's this used for?

Reply:
its a dependency of the [mock lib](https://github.com/temporalio/ui/pull/2623/files#diff-46e7b151aa03f35df855459d9c92526f2bb861deb0cfac6df195b4fe9aa844efR14)

Reply:
One of the nice things about go is that there are no dev/prod deps. Packages are included based on what application packages are included. when using `_test` on a test file, that file is not included in the compiled binary. In turn, none of its dependencies are included either. 

So the mock lib and all its deps, while part of the sum, are not included in the compiled binary.